k6/spike.js

```diff
@@ -0,0 +1,49 @@
+import { check, sleep } from 'k6';
+import exec from 'k6/execution';
+import http from 'k6/http';
+
+export const options = {
+	// Key configurations for spike in this section
+	stages: [
+		{ duration: '2m', target: 2000 }, // fast ramp-up to a high point
```

Comment:
😰 That's a lot of users!

Seems unrealistic, even if our site is under attack - because then AWS firewall will kick in to rate-limit requests. I'd suggest starting with say 500, and seeing if it breaks. Even 500 seems a lot, but at least it's testing an _unexpectedly_ high load.

Reply:
Will bring down, I was playing with numbers yesterday and found local breaks around 5000 ish if the uptake is 2m

Reply:
Well, I'm fine with you leaving this at 2000, though of course when we want to run tests against remote, we could _possibly_ hit AWS rate-limiting... Let's see!